spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -243,6 +245,15 @@ public AppRegistryController appRegistryController(
 					metadataResolver, appRegistryFJPFB, streamDefinitionService, appRegistrationAssemblerProvider);
 		}
 
+		@Bean
+		public TaskCtrController tasksCtrController(ApplicationConfigurationMetadataResolver metadataResolver,
```

Comment:
Would this fit better in the TaskEnabledConfiguration of the DataFlowControllerAutoConfiguration?   In that the CTR will not be active if tasks are not active.   Thoughts?

Reply:
Yeah, you're right.